pkg/scripts/download.go

```diff
@@ -44,14 +44,14 @@ func NewDownloader(proxyConfig *api.ProxyConfig) Downloader {
 // Returns information a boolean flag informing whether any download/copy operation
 // happened and an error if there was a problem during that operation
 func (d *downloader) Download(url *url.URL, targetFile string) (*api.SourceInfo, error) {
-	schemeReader := d.schemeReaders[url.Scheme]
+	r := d.schemeReaders[url.Scheme]
```

Comment:
> Why not reader BTW?

Copying the contents of the link I posted above:

> The basic rule: the further from its declaration that a name is used, the more descriptive the name must be. For a method receiver, one or two letters is sufficient. **Common variables such as** loop indices and **readers** can be a single letter (i, **r**). More unusual things and global variables need more descriptive names.

I don't think `reader.Read(url)` in this context is any better than `r.Read(url)`.